packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -222,5 +222,8 @@ export const messages = {
   error_push_or_deploy_on_save_no_default_username:
     'Error running push or deploy on save: No default org is set. Run "SFDX: Create a Default Scratch Org" or "SFDX: Authorize an Org", then push or deploy the changes that you just saved. Or, to disable push or deploy on save, set "salesforcedx-vscode-core.push-or-deploy-on-save.enabled" to false in your user or workspace settings for VS Code.',
   error_source_path_not_in_package_directory_text:
-    'Error deploying or retrieving source: The file or directory that you tried to deploy or retrieve isn\'t in a package directory that\'s specified in your sfdx-project.json file. Add this location to your "packageDirectories" value, or deploy or retrieve a different file or directory. For details about sfdx-project.json, see: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_ws_config.htm'
+    'Error deploying or retrieving source: The file or directory that you tried to deploy or retrieve isn\'t in a package directory that\'s specified in your sfdx-project.json file. Add this location to your "packageDirectories" value, or deploy or retrieve a different file or directory. For details about sfdx-project.json, see: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_ws_config.htm',
+  error_retrieving_previous_authorizations:
+    'Error retrieving previous authorizations. Run "SFDX: Authorize an Org" or "SFDX: Create a Default Scratch Org"',
+  missing_default_org: 'No Default Org Set'
 };
```

Comment:
@ruthemmanuelle Hi Ruth, this text will show up if the user has no previous authorizations set and tries to access the authorization list. Let me know if you'd like any changes, thanks!

Reply:
Hi @AnanyaJha . How will these two new messages be displayed?

Reply:
The `org_select_text` will be the placeholder in the text box for the drop down, which allows them to select an org or scratch org. The `missing_default_org` will display in the bottom status bar when there is no default org set for the project.